tests/manual/forwardrenderer_scene/controllerwidget.ui

```diff
@@ -292,6 +292,76 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="QGroupBox" name="dofGroupBox">
```

Comment:
this is checked by default (at startup) but the effect is off

Reply:
Fixed